modules/analyzing-the-collector-pod-status.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: PROCEDURE
+[id="analyzing-the-collector-pod-status_{context}"]
+= Analyzing the Collector pod status
+
+Examining the pod’s most recent status is another easy way to determine the cause of a Collector crash. Failure messages are recorded to the most recent status and are accessible using the `kubectl` or `oc` command:
+
+[source,terminal]
+----
+# Substitute your collector pod into this command
+$ kubectl describe pod -n stackrox collector-vclg5
+----
+
```

Comment:
```suggestion
[source,terminal]
----
$ oc describe pod -n stackrox <collector-pod-name> <1>
----
<1> If you use Kubernetes, enter `kubectl` instead of `oc`.
```

Reply:
**issue:** We don't use comments in code blocks and use `<xyz>` for listing a substitution item. 

**nitpick:** for `oc` and `kubectl` this is how we specify them using code comments.

Reply:
@agantony **nitpick:** We'll have to replace the name of the collector from the commands and instead use <xyz> for listing a substitution item. 

- Please add a substitution string such as `<collector-name>` 
- Add a code comment saying replace `<collector-name>` with the name of your collector pod.

Reply:
Done